scripts/mk.js

```diff
@@ -3870,6 +3872,7 @@ var fLineScale = 0;
 // setup main container
 var oContainers = [document.createElement("div")];
 oContainers[0].className = "game-container";
+oContainers[0].id = "game-container"
```

Comment:
You don't need this id, `oContainers[0]` is already the same as `document.getElementById("game-container")`

Reply:
thought so, just felt like we needed an id but that also works :)